src/Consolonia.Gallery/View/MainWindow.axaml

```diff
@@ -7,6 +7,7 @@
         d:DesignWidth="800"
```

Comment:
Help dialog should have a border and close button 
<img width="1494" height="1000" alt="image" src="https://github.com/user-attachments/assets/cd82fa89-b263-4509-ac3c-167de051a0b2" />

Reply:
This one I could not solve. I guess we need to review this after re-structuring ManagedWindows